packages/core/src/components/sections/ProductShelf/Overrides.tsx

```diff
@@ -0,0 +1,10 @@
+import DefaultProductCard from 'src/components/product/ProductCard'
```

Comment:
Change this to import the ProductShelf component

Reply:
It is! I was confused. Can you change the name to ProductCard instead of DefaultProductCard?